build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -48,6 +48,9 @@
     "@llvm-project//mlir:StandardToSPIRVConversions": [
         "MLIRStandardToSPIRVTransforms"
     ],
+    "@llvm-project//mlir:GPUToVulkanTransforms": [
+        "MLIRGPUtoVulkanTransforms"
```

Comment:
We should fix this name upstream. The "To" should be capitalized. In https://reviews.llvm.org/rGcd93906f93e6ff8c6619e0551b86a55ff3f4eafc I did the same for MLIRGPUtoSPIRVTransforms. You can just work around it here for right now though.

Reply:
I just sent out https://reviews.llvm.org/D82840.

Reply:
Thanks

Reply:
Obsolete as soon as LLVM is bumped :) Thanks for landing the patch.

Reply:
Thanks for fixing the naming Marius!